sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -626,11 +640,25 @@ void gnrc_sixlowpan_iphc_recv(gnrc_pktsnip_t *sixlo, void *rbuf_ptr,
         rbuf->super.current_size += (uncomp_hdr_len - payload_offset);
 #ifdef MODULE_GNRC_SIXLOWPAN_FRAG_VRB
         if (vrbe != NULL) {
+            int res = -1;
             DEBUG("6lo iphc: found route, trying to forward\n");
             ipv6_hdr->hl--;
             vrbe->super.current_size = rbuf->super.current_size;
-            /* TODO: forward here according to forwarding scheme here */
```

Comment:
Just leave the comment but drop the TODO - makes it easier to understand what's going on.

Reply:
Done